src/components/PublicationRequestFormSections/LinkAgreementForm/LinkAgreementForm.js

```diff
@@ -35,12 +35,17 @@ const LinkAgreementForm = () => {
     change('agreement.remoteId', a.id);
   };
 
+  const handleAgreementRemoved = () => {
+    setAgreement({});
+    change('agreement.remoteId', undefined);
+  };
+
   return (
     <Accordion
       label={<FormattedMessage id="ui-oa.publicationRequest.agreement" />}
     >
-      <Row>
-        <Col xs={3}>
+      <Row between="xs">
```

Comment:
Did you verify what this actually does and whether it's necessary?

Reply:
It looks like it evenly spaces the columns? We shouldn't be doing that, instead I think we should pick the column sizes manually:

```
<Row>
  <Col xs={3}>
    ...
  </Col>
  <Col xs={3}>
    ...
  </Col>
  <Col xs={3}>
    ...
  </Col>
</Row>
```
etc